include/mega/crypto/sodium.h

```diff
@@ -19,7 +19,6 @@
  * program.
  */
 
-#ifdef USE_SODIUM
```

Comment:
I added the line after the header file macro `SODIUM_H`. Not sure if it would cause any interference during the building process.